RPIPs/RPIP-60.md

```diff
@@ -52,6 +54,7 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
   - This is mitigated by the pDAO's ability to replace the security council. While a valid proposal could be blocked once, it could not be blocked indefinitely.
 - The veto process can itself be modified by contract upgrades. However, if a corrupt proposal tries to change the veto process, that can be defended against by using the veto process.
 - The security council veto means that if pDAO governance were captured, protocol upgrades could be stopped at will (by installing a compromised security council). The oDAO would be unable to pass any contract upgrades (without the approval of the compromised security council), and the only way back would be to recapture pDAO governance. While this means there is a real tradeoff, this "inability to upgrade" is seen as less problematic than the "ability to pass any upgrade" that a rogue oDAO would have without the veto.
+- The quorum for this veto is lower than for security council proposals. This is intended to provide additional resilience to, eg, a rogue oDAO that attempts to bribe a security council into not vetoing a corrupt proposal (as fewer bribe-resistant members are needed). This sort of interaction doesn't exist for normal security council proposals, which is what leads to a different desired threshold. 
```

Comment:
```suggestion
- The quorum for this veto is lower than for security council proposals. This is intended to provide additional resilience to, eg, a rogue oDAO that attempts to bribe a security council into not vetoing a corrupt proposal (as fewer bribe-resistant members are needed).
```
I'm not sure that last statement is true? And it seems difficult to prove the negative. Maybe just remove? Stylistically its also providing rationale on something not within this RPIP (normal security council proposals)

Reply:
Also feel free to ignore.

Reply:
Happy to delete that line. Was trying to provide a reason for not using the existing `rocketDAOProtocolSettingsSecurity.members.quorum` setting

Reply:
Ah, okay. Maybe reword first sentence to better capture this? 'The quorum for this veto is lower than for security council proposals set in rocketDAOProtocolSettingsSecurity.members.quorum'